requests.go

```diff
@@ -75,18 +74,18 @@ func (lk *logKeeper) checkContentLength(r *http.Request) *apiError {
 
 type ctxKey int
 
-const (
-	requestCtxKey ctxKey = iota
-)
+const requestCtxKey ctxKey = iota
```

Comment:
Does it actually make sense to set a single constant to iota?

Reply:
not at all, but it's reasonable.